src/components/gameGomponent/gameComponent.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+class Game extends Component {
+   render() {
+       return (
+           <div>
+               <div className="Game-header">
```

Comment:
Can we extract this into a `GameHeader` component?

Reply:
Working on it now

Reply:
Extracted into a GameHeader Component